src/optimizer/filter_pullup.cpp

```diff
@@ -0,0 +1,90 @@
+#include "duckdb/optimizer/filter_pullup.hpp"
+#include "duckdb/planner/operator/logical_join.hpp"
+
+namespace duckdb {
+using namespace std;
```

Comment:
Could you remove all the `using namespace std`, please? Hannes has just zapped all of them (#1233) and we would prefer to keep these out.